Feat/street view panorama examples (#101)
ubilabs/google-maps-react-hooks: examples/street-view-panorama-map/README.md

# `useStreetViewPanorama` Hook Setup Example

> This is an example setup of **Street View Panorama** to show the usage of the **useStreetViewPanorama hook** without passing a div element with the Google Maps React Hooks library.

Reviewer: I think most of the readers won't know what "without passing a div element" means in this case.
```suggestion
This is an example setup of **Street View Panorama** to show the usage of the **useStreetViewPanorama hook** to enable Street View in an existing map with the Google Maps React Hooks library.
```

Author: Yes sounds good 👍

## Instructions

To run this project, clone the Google Maps React Hooks repository locally.

First go to the root of the repository and run:

```shell
npm install
````

once to install all dependencies.

Then start this example locally with

```shell
npm run start:street-view-panorama-map-example
```

**NOTE**:
To see the examples it is needed to add an `.env` file with a [Google Maps API key](https://developers.google.com/maps/documentation/embed/get-api-key#:~:text=Go%20to%20the%20Google%20Maps%20Platform%20%3E%20Credentials%20page.&text=On%20the%20Credentials%20page%2C%20click,Click%20Close.) in the following format:

`GOOGLE_MAPS_API_KEY="<YOUR API KEY HERE>"`

An example can be found in `.env.example`.

## Output

The project will start at [localhost:1234](http://localhost:1234) and show a map with street view.

> - Add image here

Reviewer: ?
